.github/workflows/python_tests.yml

```diff
@@ -1,35 +1,51 @@
-name: lint_python
+# https://docs.github.com/en/actions/guides/creating-postgresql-service-containers
+name: python_tests
 on: [pull_request, push]
 jobs:
-  lint_python:
+  python_tests:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9]  # 2.7
+        python-version: [3.9]
     runs-on: ubuntu-latest
+    services:
+      # Label does not set the Postgres host name which will be localhost
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_USER: runner
+          POSTGRES_HOST_AUTH_METHOD: trust
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Map TCP port 5432 on service container to the host
+          - 5432:5432
     steps:
       - uses: actions/checkout@v2
-      - uses: harmon758/postgresql-action@v1
-        with:
-          postgresql version: '13'  # See https://hub.docker.com/_/postgres for available versions
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - run: pip install bandit black codespell flake8 isort mypy psycopg2 pytest pyupgrade
-      - run: bandit -r . || true
+      - run: pip install black codespell flake8 isort mypy pytest pyupgrade safety
```

Comment:
We don't want safety running here; our CI should be deterministic, and not fail due to factors unrelated to our code (i.e. a dependency getting a security vulnerability). Dependabot should be catching dependency security issues, no?

```suggestion
      - run: pip install black codespell flake8 isort mypy pytest pyupgrade
```